mailer.js

```diff
@@ -61,8 +67,8 @@ module.exports = function (log) {
     this.mailer.close()
   }
 
-  Mailer.prototype._supportLinkAttributes = function () {
-    return linkAttributes(this.supportUrl)
+  Mailer.prototype._supportLinkAttributes = function (template) {
```

Comment:
I found the variable name `template` in these methods confusing, at first I was unsure if `template` was the template name or the actual template. Could you rename the variable to `templateName` or similar, where it's used as a name, to help with clarity?